src/dos/dos_locale.cpp

```diff
@@ -553,14 +553,21 @@ static const std::string AnsiReset     = "[reset]";
 static const std::string AnsiWhite     = "[color=white]";
 static const std::string AnsiHighlight = "[color=light-green]";
```

Comment:
Maybe it would be nice to make these predefined in `ansi_markup.h` for all colours.

Reply:
OK, added, in a separate commit.

Eventually we should rework the whole code to use these definitions - but this is a separate task.